Sources/Extensions/SwiftStdlib/Deprecated/SwiftStdlibDeprecated.swift

```diff
@@ -39,6 +39,20 @@ public extension Bool {
 
 extension String {
 
+	#if canImport(Foundation)
+	/// SwifterSwift: Check if string is valid email format.
+	/// **Note that this property does not validate the email address against an email server. It merely attempts to determine whether its format is suitable for an email address.**.
+	///
+	///		"john@doe.com".isEmail -> true
+	///
+	@available(*, deprecated: 4.5.0, message: "Use isValidEmail instead", renamed: "isValidEmail")
+	public var isEmail: Bool {
```

Comment:
Why did you move this to a different extension?

Reply:
I'm guessing to deprecate the old API

Reply:
@guykogus We move the deprecate extensions to those files to exclude them for coverage reports :))

Reply:
Yeah it is an API breaking change that's not backwards compatible. We're not following semver properly here

Reply:
I really need to read more carefully the name of the file! 😅

Reply:
btw, I think since we used `renamed`, Xcode will suggest replacing the deprecated function with one click 😃